### PR TITLE
Fix openApi when no data provided

### PR DESCRIPTION
**Pull request by Te4g**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tickets       | 
| License       | MIT
| Doc PR        | 

Capture d’écran 2020-10-13 à 22 46 14

Found this bug on a brand new Symfony project while trying to check the Open Api documentation while no Entity was created yet with `"api-platform/core": "2.6.x-dev"`

### CODE REVIEW

File: src/OpenApi/Model/Paths.php [hunk 1]
```diff
@@ -27,7 +27,7 @@ public function getPath(string $path): ?PathItem
         return $this->paths[$path] ?? null;
     }
 
-    public function getPaths(): array
+    public function getPaths(): ?array
     {
         return $this->paths;
```
Comment (dunglas) [inline src/OpenApi/Model/Paths.php:32]: Isn't it possible to initialize the property to an empty array instead?

**Comment by soyuka** [comment]
Paths is required ([ref](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#fixed-fields)) but may be empty, please use an empty array as suggested.

File: src/OpenApi/Serializer/OpenApiNormalizer.php [hunk 2]
```diff
@@ -56,7 +56,7 @@ private function recursiveClean($data): array
             }
 
             // Side effect of using getPaths(): Paths which itself contains the array
-            if ('paths' === $key) {
+            if ('paths' === $key && isset($data['paths']['paths'])) {
                 $value = $data['paths'] = $data['paths']['paths'];
```
Comment (soyuka) [inline src/OpenApi/Serializer/OpenApiNormalizer.php:60]:
```suggestion
                $value = $data['paths'] = $data['paths']['paths'] ?? [];
```

File: src/OpenApi/Model/Paths.php [hunk 3]
```diff
@@ -27,7 +27,7 @@ public function getPath(string $path): ?PathItem
         return $this->paths[$path] ?? null;
     }
 
-    public function getPaths(): array
+    public function getPaths(): ?array
```
Comment (soyuka) [inline src/OpenApi/Model/Paths.php:30]: revert this change

Comment (Te4g) [reply src/OpenApi/Model/Paths.php:30]: 👍

File: src/OpenApi/Serializer/OpenApiNormalizer.php (hunk 2 above)
Comment (Te4g) [reply src/OpenApi/Serializer/OpenApiNormalizer.php]: This is not working, we still have the array type error from the `getPaths` method, i just made a new proposal :)

File: src/OpenApi/Model/Paths.php (hunk 1 above)
Comment (Te4g) [reply src/OpenApi/Model/Paths.php:32]: Sure 👌

**Review by dunglas — APPROVED** [review]

**Comment by soyuka** [comment]
Perfect thanks @Te4g
